tango_simlib/examples/override_class.py

```diff
@@ -729,6 +731,26 @@ def action_setstowmode(self, model, tango_dev=None, data_input=None):
                                    "DISH is not in {} mode.".format(_allowed_modes),
                                    "SetStowMode()",
                                    ErrSeverity.WARN)
+        try:
+            pointing_state_quant = model.sim_quantities['pointingState']
+        except KeyError:
+            Except.throw_exception(
+                "DISH Command Failed",
+                "The quantity 'pointingState' is not in the Dish model.",
+                "Slew()", ErrSeverity.WARN)
+
+        pointing_state_enum_val = pointing_state_quant.last_val
+        pointing_state_str_val = (
+            pointing_state_quant.meta['enum_labels'][int(pointing_state_enum_val)])
+        if pointing_state_str_val != 'SLEW':
```

Comment:
- [ ] Should be STOW? Also don't you want to allow re-setting of STOW even if already stowing?